### PR TITLE
#automate_report_submission: Add LD flag in expense form and tasks page

### PR DESCRIPTION
**Pull request by yksolanki9**
Changes in this PR
- Added LD support
- Content change in add to report list view - Removed state pill and updated text

New design
localhost_8100_enterprise_add_edit_expense;navigate_back=true(iPhone SE) (2)

### CODE REVIEW

File: src/app/core/services/launch-darkly.service.ts [hunk 1]
```diff
@@ -62,6 +62,10 @@ export class LaunchDarklyService {
     return this.getVariation('keyboard_plugin_enabled', true);
   }
 
+  checkIfReportAutoSubmissionIsEnabled() {
```
Comment (Dimple16) [inline src/app/core/services/launch-darkly.service.ts:65]: lets have the method name same as the key -> `checkIfAutomateReportSubmissionIsEnabled`

Comment (Dimple16) [reply src/app/core/services/launch-darkly.service.ts:65]: also, why do we have the key with `_enabled`? isn't this incorrect? `automate_report_submission_enabled` this should have been `automate_report_submission`

**Review by Dimple16 — CHANGES_REQUESTED** [review]

File: src/app/core/services/launch-darkly.service.ts (hunk 1 above)
Comment (yksolanki9) [reply src/app/core/services/launch-darkly.service.ts:65]: Yes, `enabled` does sound a bit redundant here, but this is being used across all clients and will be removed when we release this feature for all orgs, so shouldnt be an issue

Comment (yksolanki9) [reply src/app/core/services/launch-darkly.service.ts]: Renamed the method
\

**Review by Dimple16 — APPROVED** [review]

**Pull request merged**